lib/function.js

```diff
@@ -318,6 +318,43 @@ const builtInsImpls = {
     valueOf: (target, context, options, runState) => {
       return target.valueOf();
     },
+    includes: (target, context, options, runState) => {
+      const elem = context.get('searchElement');
+      const from = context.get('fromIndex');
+      return target.includes(elem, from);
+    },
+    match: (target, context, options, runState) => {
+      const pattern = context.get('pattern');
+      return target.match(pattern);
+    },
+    matchAll: (target, context, options, runState) => {
+      const pattern = context.get('pattern');
+      return target.match(pattern);
+    },
+    replace: (target, context, options, runState) => {
+      const searchValue = context.get('searchValue');
+      const newValue = context.get('newValue');
+      return target.replace(
+        searchValue, 
+        newValue instanceof AbstractFunction ? 
+        (x) => newValue.exec(x, context, options, runState) :
+        newValue
+      )
+    },
+    replaceAll: (target, context, options, runState) => {
+      const searchValue = context.get('searchValue');
+      const newValue = context.get('newValue');
+      return target.replaceAll(
+        searchValue, 
+        newValue instanceof AbstractFunction ? 
+        (x) => newValue.exec(x, context, options, runState) :
```

Comment:
`exec` method requires the first argument to be an array of parameters. Please fix the code to pass in all the arguments to the replacer function.